app/routes/_index/route.tsx

```diff
@@ -1,5 +1,6 @@
 import { LinksFunction, LoaderFunctionArgs } from '@remix-run/node';
 import { Link, MetaFunction, useLoaderData, useNavigate, json } from '@remix-run/react';
+import { products as wixStoresProducts } from '@wix/stores';
```

Comment:
is this our common pattern to write like that when using wix apis ?
it's just I never saw such style in this repo

Reply:
well.. in this route `products` name is taken in code, so I renamed import.. I believe it's a normal practice in general